docs/jazzy.yml

```diff
@@ -22,68 +22,108 @@ custom_categories:
       - NavigationViewController
       - NavigationViewControllerDelegate
       - RouteVoiceController
+      - VoiceControllerDelegate
+      - MapboxVoiceController
+      - NavigationOptions
+      - NavigationView
   - name: Core Navigation Logic
     children:
       - NavigationService
+      - NavigationServiceDelegate
       - MapboxNavigationService
+      - Router
+      - RouterDataSource
       - RouteController
-      - RouteControllerDelegate
+      - RouterDelegate
       - RouteProgress
       - RouteLegProgress
       - RouteStepProgress
-      - RouteControllerNotificationUserInfoKey
+      - NavigationEventsManager
+      - EventsManagerDataSource
+      - NavigationLocationManager
+      - ReplayLocationManager
+      - SimulatedLocationManager
+      - NavigationRouteOptions
+      - NavigationMatchOptions
+      - NavigationDirections
+      - SimulationMode
   - name: Maps
     children:
       - NavigationMapView
       - NavigationMapViewDelegate
       - MGLMapView
   - name: Styling
     children:
+      - LanesStyleKit
+      - ManeuversStyleKit
       - Style
-      - StyleKitArrows
-      - StylableButton
       - StylableLabel
+      - StylableView
+      - StylableTextView
+      - StylableButton
+      - StyleManager
+      - StyleManagerNotificationUserInfoKey
+      - StyleManagerDelegate
+      - DayStyle
+      - NightStyle
   - name: Formatters
     children:
       - DistanceFormatter
-      - RouteStepFormatter
   - name: UI Components
     children:
-      - RouteTableViewController
-      - NavigationPulleyViewController
       - Button
-      - CellSubtitleLabel
-      - CellTitleLabel
       - DashedLineView
       - HighlightedButton
       - LineView
       - ProgressBar
       - SeparatorView
       - SubtitleLabel
       - TitleLabel
-      - TurnArrowView
+      - ManeuverView
```

Comment:
Oh wow, this class got renamed a long time ago in #745. 😅 ManeuverView is marked `:nodoc:`, but we should actually document it, because developers may need to adjust its `appearance()` when creating a new Style. Let’s go with this description for now:

> A view that contains a simple image indicating a type of maneuver.